Code/Mantid/Framework/Algorithms/inc/MantidAlgorithms/MonitorEfficiencyCorUser.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MANTID_ALGORITHMS_MONITOREFFICIENCYCORUSER_H_
+#define MANTID_ALGORITHMS_MONITOREFFICIENCYCORUSER_H_
+
+#include "MantidKernel/System.h"
+#include "MantidAPI/Algorithm.h"
+
+namespace Mantid
+{
+  namespace Algorithms
+  {
+
+class DLLExport MonitorEfficiencyCorUser : public API::Algorithm
+{
+public:
+  /// (Empty) Constructor
+  MonitorEfficiencyCorUser();// : Mantid::API::Algorithm() {}
+  /// Virtual destructor
+  virtual ~MonitorEfficiencyCorUser();
+  /// Algorithm's name
+  virtual const std::string name() const { return "MonitorEfficiencyCorUser"; }
+  ///Summary of algorithms purpose
+  virtual const std::string summary() const {return "This algorithm normalises the counts by the monitor counts with additional efficiency correction according to the formula set in the instrument parameters file.";}
```

Comment:
We tend to use US spelling. You've used Normalizes in the rst. Could you please change this to normalizes.